README.md

```diff
@@ -12,6 +12,11 @@ To see all available configuration flags:
 
     ./pgbouncer_exporter -h
 
+## Run with Docker
+
+```
+docker run chriscowley/pgbouncer_exporter -pgBouncer.connectionString=postgres://postgres:@pgbouncer:6432/pgbouncer?sslmode=disable
```

Comment:
Do we need to update this line?

Reply:
It is dependent on how the user's environment, so potentially yes

I have reverted back to using Alpine as the final base image. Building with Alpine (and thus musl) then runnning on busybox (ulibc iirc) was not working.